Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/26.0/InspectorBackendCommands.js

```diff
@@ -0,0 +1,571 @@
+InspectorBackend.registerCommand("Canvas.setRecordingAutoCaptureFrameCount", null, [{"name": "count", "type": "number"}], []);
+InspectorBackend.registerCommand("Canvas.startRecording", null, [{"name": "canvasId", "type": "string"}, {"name": "frameCount", "type": "number", "optional": true}, {"name": "memoryLimit", "type": "number", "optional": true}], []);
+InspectorBackend.registerCommand("Canvas.stopRecording", null, [{"name": "canvasId", "type": "string"}], []);
+InspectorBackend.registerCommand("Canvas.requestShaderSource", null, [{"name": "programId", "type": "string"}, {"name": "shaderType", "type": "string"}], ["source"]);
+InspectorBackend.registerCommand("Canvas.updateShader", null, [{"name": "programId", "type": "string"}, {"name": "shaderType", "type": "string"}, {"name": "source", "type": "string"}], []);
+InspectorBackend.registerCommand("Canvas.setShaderProgramDisabled", null, [{"name": "programId", "type": "string"}, {"name": "disabled", "type": "boolean"}], []);
+InspectorBackend.registerCommand("Canvas.setShaderProgramHighlighted", null, [{"name": "programId", "type": "string"}, {"name": "highlighted", "type": "boolean"}], []);
+InspectorBackend.registerEvent("Canvas.canvasAdded", null, ["canvas"]);
+InspectorBackend.registerEvent("Canvas.canvasRemoved", null, ["canvasId"]);
+InspectorBackend.registerEvent("Canvas.canvasSizeChanged", null, ["canvasId", "width", "height"]);
+InspectorBackend.registerEvent("Canvas.canvasMemoryChanged", null, ["canvasId", "memoryCost"]);
+InspectorBackend.registerEvent("Canvas.extensionEnabled", null, ["canvasId", "extension"]);
+InspectorBackend.registerEvent("Canvas.clientNodesChanged", ["page"], ["canvasId"]);
+InspectorBackend.registerEvent("Canvas.recordingStarted", null, ["canvasId", "initiator"]);
+InspectorBackend.registerEvent("Canvas.recordingProgress", null, ["canvasId", "frames", "bufferUsed"]);
+InspectorBackend.registerEvent("Canvas.recordingFinished", null, ["canvasId", "recording"]);
+InspectorBackend.registerEvent("Canvas.programCreated", null, ["shaderProgram"]);
+InspectorBackend.registerEvent("Canvas.programDeleted", null, ["programId"]);
+InspectorBackend.registerCanvasDispatcher = InspectorBackend.registerDispatcher.bind(InspectorBackend, "Canvas");
+InspectorBackend.activateDomain("Canvas", ["page", "web-page"]);
+
+// Console
+InspectorBackend.registerDomain("Console", ["itml", "javascript", "page", "service-worker", "worker"]);
+InspectorBackend.registerEnum("Console.ChannelSource", {XML: "xml", JavaScript: "javascript", Network: "network", ConsoleAPI: "console-api", Storage: "storage", Appcache: "appcache", Rendering: "rendering", CSS: "css", Accessibility: "accessibility", Security: "security", ContentBlocker: "content-blocker", Media: "media", MediaSource: "mediasource", WebRTC: "webrtc", ITPDebug: "itp-debug", PrivateClickMeasurement: "private-click-measurement", PaymentRequest: "payment-request", Other: "other"});
```

Comment:
oh hmm apparently `MessageSource::AppCache` is still in use.  i think we may need to add this back to all the old protocol `.js`/`.json` in that case (i.e. undo these changes from [289820@main](https://commits.webkit.org/289820@main))

Reply:
AppCache has been marked for deprecation in Safari since 2018:
https://bugs.webkit.org/show_bug.cgi?id=181764

Both Chrome and Firefox have deprecated the feature:
https://web.dev/articles/appcache-removal

We intentionally rewrote the older protocol snapshots to remove AppCache to prevent issues where Web Inspector doesn't have the frontend required to inspect an older device that happens to still use the deprecated feature: https://github.com/WebKit/WebKit/pull/39648#discussion_r1933144397

Reply:
@AnujPanta1 there are some references to `appcache` that should be manually removed, like we did for the [previous snapshot patch](https://github.com/WebKit/WebKit/pull/41071#discussion_r1966341561)

Reply:
PR to replace `MessageSource::AppCache`: https://github.com/WebKit/WebKit/pull/47122